### ISSUE TITLE
Unable to use in latest beta 2021.8.0b1

### ISSUE THREAD

**Issue by aidbish** [elad-bar/ha-blueiris#120]
the component is unable to be setup in latest beta

Logger: homeassistant.setup
Source: setup.py:311
First occurred: 9:38:40 PM (1 occurrences)
Last logged: 9:38:40 PM

Unable to prepare setup for platform blueiris.binary_sensor: Platform not found (cannot import name 'Message' from 'homeassistant.components.mqtt' (/usr/src/homeassistant/homeassistant/components/mqtt/__init__.py)).

Logger: custom_components.blueiris.managers.entity_manager
Source: custom_components/blueiris/managers/entity_manager.py:577
Integration: Blue Iris NVR (documentation)
First occurred: 9:38:40 PM (3 occurrences)
Last logged: 9:39:41 PM

Failed to update, step: Start updating domain binary_sensor, Error: 'binary_sensor', Line: 197

**Comment by Airey001** [comment]
Can verify still an issue in 2021.8.0b3

**Comment by Airey001** [comment]
Logger: custom_components.blueiris.managers.entity_manager
Source: custom_components/blueiris/managers/entity_manager.py:577
Integration: Blue Iris NVR (documentation)
First occurred: 8:40:31 PM (13 occurrences)
Last logged: 8:45:49 PM

Failed to update, step: Start updating domain binary_sensor, Error: 'binary_sensor', Line: 197

**Comment by pjponymoore** [comment]
Can also verify still an issue in 2021.8.0b3

**Comment by pjponymoore** [comment]
core-2021.8.0b4

Still Issue with component

**Comment by aidbish** [comment]
its not going to change, regardless of the beta. the component needs fixing

**Comment by elad-bar** [comment]
Hi, i'm no longer maintaining that component on regular basis as I'm not using BI anymore (switched to Shinobi),
Once HA 2021.8 will become official I will try to fix it, but don't have an option to test it, will update here if you would like to test it,
An alternative solution is that someone will do a PR with the fix

Thanks

**Comment by elad-bar** [comment]
Submitted an issue to HA Core repo, there is a breaking change over there without warning:
https://github.com/home-assistant/core/issues/53758

I can release a pre-release version with the fix if you would like to test it

**Comment by elad-bar** [comment]
Pre-release: v1.0.6b1
https://github.com/elad-bar/ha-blueiris/releases/tag/v1.0.6b1

**Comment by mrdarrengriffin** [comment]
Hello! After your 1.0.6b1 release, I reinstalled the addon but after configuring the integration, it still fails:

Logger: custom_components.blueiris.managers.entity_manager
Source: custom_components/blueiris/managers/entity_manager.py:577
Integration: Blue Iris NVR (documentation)
First occurred: 16:11:04 (7 occurrences)
Last logged: 16:14:04

Failed to update, step: Start updating domain binary_sensor, Error: 'binary_sensor', Line: 197

**Comment by elad-bar** [comment]
Will recheck later today, thanks for the logs

**Comment by elad-bar** [comment]
please try again with v1.0.6b2:
https://github.com/elad-bar/ha-blueiris/releases/tag/v1.0.6b2

**Comment by pjponymoore** [comment]
Tested this morning and working perfect again

Thanks for extra effort on this.  Great work!!!!!

**Comment by aidbish** [comment]
Confirming with V1.0.6b2, it is now working with latest beta. Thanks @elad-bar for fixing even though you no longer using BI.
Out of curiosity, how does shinobi compare with BI? Does it also use Deepstack?

**Comment by elad-bar** [comment]
Thanks for updating, will make it release.

Shinobi is running in docker, which means I don't need additional Windows PC and everything is running on one PC,
As opposed to BI which I needed to run it with GPU to be able to process 16 camera, within Shinobi it's barely reach to 20% of CPU,
Since Shinobi is an open source and supports plugins, I wrote integration between Shinobi and DeepStack (Object detection and face recognition), for DeepStack I'm using GPU.
With half hardware I've reached for better performance using Shinobi and it works great!
